__tests__/integration.test.js

```diff
@@ -104,4 +104,39 @@ describe('/api/articles', () => {
         }) 
     })
 })
-
+describe('/api/articles/:article_id/comments', () => {
+    test('GET: 200, returns an array of comment objects asociated with the article ID', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            const { comments } = body;
+            expect(comments).toHaveLength(11)
+        })
+    })
+    test('GET: 200, returns an array of comment objects with the most recent comment first', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body}) => {
+            const { comments } = body;
+            expect(comments).toBeSortedBy('created_at', {descending: true})
+        })
+    })
```

Comment:
Excellent test, could cut line 120 because `expect(200)` is already in the previous test